docs/pages/using_source.md

```diff
@@ -0,0 +1,301 @@
+ETC library modules.
+
+<!-- CODE_BLOCK_START -->
+```c
+#include "sacn/source.h"
+
+// During startup:
+EtcPalLogParams log_params = ETCPAL_LOG_PARAMS_INIT;
+// Initialize log_params...
+
+etcpal_error_t init_result = sacn_init(&log_params);
+// Or, to init without worrying about logs from the sACN library...
+etcpal_error_t init_result = sacn_init(NULL);
+
+// During shutdown:
+sacn_deinit();
+```
+<!-- CODE_BLOCK_MID -->
+```cpp
+#include "sacn/cpp/source.h"
+
+// During startup:
+EtcPalLogParams log_params = ETCPAL_LOG_PARAMS_INIT;
+// Initialize log_params...
+
+etcpal::Error init_result = sacn::Init(&log_params);
+// Or, to init without worrying about logs from the sACN library...
+etcpal::Error init_result = sacn::Init();
+
+// During shutdown:
+sacn::Deinit();
+```
+<!-- CODE_BLOCK_END -->
+
+A source must first be created before it can be used to transmit sACN data. Each source has a name
+and a CID. These are specified in the source's configuration structure, along with other optional
+settings. Once the source's configuration is initialized, it can simply be passed into the source
+Create/Startup function. In this function, you can also specify specific network interfaces for the
+source to use. You can also indicate that all network interfaces should be used. If the source
+Create/Startup function succeeds, that source's functionality will become available. In C, this is
+accessed with a source handle, whereas in C++ it's accessed through a Source object. A source can
+later be destroyed using the source Destroy/Shutdown function.
+
+<!-- CODE_BLOCK_START -->
+```c
+EtcPalUuid my_cid;
+char my_name[SACN_SOURCE_NAME_MAX_LEN];
+// Assuming my_cid and my_name are initialized by the application...
+
+SacnSourceConfig my_config = SACN_SOURCE_CONFIG_DEFAULT_INIT;
+sacn_source_config_init(&my_config, &my_cid, my_name);
+
+SacnMcastInterface my_netints[NUM_MY_NETINTS];
+// Assuming my_netints and NUM_MY_NETINTS are initialized by the application...
+
+sacn_source_t my_handle;
+
+// If you want to specify specific network interfaces to use:
+etcpal_error_t result = sacn_source_create(&my_config, &my_handle, my_netints, NUM_MY_NETINTS);
+// Or, if you just want to use all network interfaces:
+etcpal_error_t result = sacn_source_create(&my_config, &my_handle, NULL, 0);
+
+// To destroy the source when you're done with it:
+sacn_source_destroy(my_handle);
+```
+<!-- CODE_BLOCK_MID -->
+```cpp
+etcpal::Uuid my_cid;
+std::string my_name;
+// Assuming my_cid and my_name are initialized by the application...
+
+Settings my_config(my_cid, my_name);
+
+std::vector<SacnMcastInterface> my_netints;
+// Assuming my_netints is initialized by the application...
+
+Source my_source;
+
+// If you want to specify specific network interfaces to use:
+etcpal::Error result = my_source.Startup(my_config, my_netints);
+// Or, if you just want to use all network interfaces:
+etcpal::Error result = my_source.Startup(my_config, std::vector<SacnMcastInterface>());
```

Comment:
As I commented on Nick's PR, the second argument here should have a default value, so it can just be
```cpp
my_source.Startup(my_config);
```